### PR TITLE
ci: Add MSI package test

### PR DESCRIPTION
**Pull request by gsanchezgavier**
#### Description of the changes

This PR adds the action to test msi packages and change the `-version` to  `-show_version` (and the format) command to be consistent to the rest of integrations.

### Author

- [x] the PR should focus on a single subject. Change only relevant files to the problem you’re working on
Clean and format the code
- [ ] add unit tests for your changes and make sure all unit tests are passing
- [ ] add a risk label after carefully considering the "blast radius" of your changes
- [x] describe the _intent_ of your changes in the description. don't just rewrite your code in prose
- [x] assign at least one reviewer
- [x] address the feedback 
- [ ] add a "ready for review" label. Only PRs with this label will be reviewed.

### Reviewer

- [ ] review code for readability
- [ ] verify that high risk behavior changes are well tested
- [ ] check license for any new external dependency
- [ ] ask questions about anything that isn't clear and obvious
- [ ] approve the PR when you consider it's good to merge

### CODE REVIEW

File: .github/workflows/prerelease.yml [hunk 1]
```diff
@@ -156,7 +163,16 @@ jobs:
         shell: pwsh
         run: |
           build\windows\package_msi.ps1 -integration "$env:INTEGRATION" -arch ${{ matrix.goarch }} -tag "$env:TAG" -pfx_passphrase "$env:PFX_PASSPHRASE" -pfx_certificate_description "$env:PFX_CERTIFICATE_DESCRIPTION"
+      - name: Test win packages installation
+        uses: paologallinaharbur/test-packages-action/windows@add_win_test
```
Comment (gsanchezgavier) [inline .github/workflows/prerelease.yml:167]: This needs to be changed to the final repository/version before merge

Comment (arvdias) [reply .github/workflows/prerelease.yml:167]: Agreed. I think the opensource group already has a repo for actions, at least form what I can tell from the repolinter action, they use: **newrelic/repolinter-action@v1**

File: .github/workflows/prerelease.yml [hunk 2]
```diff
@@ -156,7 +163,16 @@ jobs:
...
+        with:
+          tag: ${{ env.TAG }}
+          integration: nri-${{ env.INTEGRATION }}
+          arch: ${{ matrix.goarch }}
+          upgrade: ${{ matrix.test-upgrade }}
       - name: Upload MSI to GH
+        # To avoid upload packages twice
+        if: startsWith(matrix.test-upgrade, 'false')
```
Comment (arvdias) [inline .github/workflows/prerelease.yml:175]: does it need to be "startsWith"? just curious,  if testing for "equal" wouldn't work

**Comment by arvdias** [comment]
I would try to move the GPG_* and PFX_* env vars to the specific action they are needed. That way we can use the secrest where they are needed instead of being declared globally. It's not a huge issue, but it's a bit cleaner and perhaps a bit more secure

File: .github/workflows/prerelease.yml (hunk 2 above)
Comment (gsanchezgavier) [reply .github/workflows/prerelease.yml:175]: my first approach was to use `==` but is [was not working ](https://github.com/newrelic/nri-vsphere/runs/1763684685?check_suite_focus=true) . I don't [see](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions) a function `equal` that would be clearer i agree.
I based on [this](https://github.community/t/matrix-cannot-be-used-in-jobs-level-if/17177/3) from the community

**Review by arvdias — APPROVED** [review]
lgtm

**Pull request merged**